drivers/net/ethernet/intel/idpf/eth/idpf_ethtool.c

```diff
@@ -387,7 +422,7 @@ static int idpf_set_ringparam(struct net_device *netdev,
 		goto unlock_mutex;
 	}
 
-	config_data = &vport->adapter->vport_config[idx]->user_config;
+	config_data = &vport->adapter->vport_config->user_config;
```

Comment:
IF this expression is occurring frequently : vport->adapter->vport_config->user_config 

then again  I suggest writing a macro and using it consistently.

Reply:
Done!